src/main/java/Task.java

```diff
@@ -0,0 +1,74 @@
+import java.util.ArrayList;
+
+public class Task {
+    protected String taskDescription;
+    protected boolean isDone;
+    private static int taskNumber = 0;
+    private static int numberOfCompletedTasks =0;
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static int completedTaskCounter =0;
+    private static Task newestTask;
+
+    public Task(String task) {
+        this.taskDescription = task;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[\u2713]" : "[\u2718]"); //return tick or X symbols
```

Comment:
As the reviewer above has said, you could consider using named constants in place of your Unicode symbols to make it easier to understand what your code will output.